pyigm/cgm/utils.py

```diff
@@ -15,73 +15,228 @@
 from linetools import utils as ltu
 
 from pyigm.field.galaxy import Galaxy
+from pyigm.abssys.igmsys import IGMSystem
 
 
-def calc_rho(galaxy, igm_sys, cosmo, ang_sep=None, correct_lowz=True,
-             Galactic=False, d_Sun=8.0*u.kpc):
+def calc_cgm_rho(galaxy, igm_sys, cosmo, ang_sep=None, correct_lowz=True):
     """ Calculate the impact parameter between the galaxy and IGM sightline
+    Mainly a wrapper to pyigm.utils.calc_rho
 
     Parameters
     ----------
-    galaxy : Galaxy object
-    igm_sys : IGMSystem object or list
+    galaxy : Galaxy
+    igm_sys : IGMSystem or list
     cosmo : astropy.cosmology
-    Galactic : bool, optional
-      Calculate for our Galaxy!
+    ang_sep : Quantity, optional
     correct_lowz : bool, optional
-      Apply corrections for the local universe, as desired
-      Follows Mould et al. 2000
-    ang_sep : Angle or Quantity
-      Input angular separation
-      May speed up calculation
```

Comment:
Why remove all this?  Some potentially very useful information here...

Reply:
Decided to use **kwargs instead